### PR TITLE
Support custom_update property coming from PUE data

### PR DESCRIPTION
**Pull request by sc0ttkclark**
https://central.tri.be/issues/94029

To support custom icons property (new property) as well as other future ones as necessary

### CODE REVIEW

File: src/Tribe/PUE/Utility.php [hunk 1]
```diff
@@ -104,6 +107,19 @@ public function to_wp_format() {
 				$update->upgrade_notice = $this->upgrade_notice;
 			}
 
+			// Support custom $update properties coming straight from PUE
+			if ( ! empty( $this->custom_update ) ) {
+				$custom_update = get_object_vars( (object) $this->custom_update );
```
Comment (bordoni) [inline src/Tribe/PUE/Utility.php]: Why cast as Object than use `get_object_vars()`?

**Comment by sc0ttkclark** [comment]
PR itself works great: http://sc0tt.me/1N1o220w1W06/Screen%20Shot%202017-12-14%20at%202.46.40%20PM.png

File: src/Tribe/PUE/Utility.php (hunk 1 above)
Comment (sc0ttkclark) [reply src/Tribe/PUE/Utility.php]: It doesn't have to be there, I can remove the cast. It comes through as an object (or nothing at all).

Comment (sc0ttkclark) [reply src/Tribe/PUE/Utility.php]: Removed the cast

**Comment by barryhughes** [comment]
https://central.tri.be/issues/95867

**Pull request merged**